samples/openapi3/client/petstore/python/tests_manual/test_request_body.py

```diff
@@ -123,7 +123,7 @@ def test_application_x_www_form_urlencoded_serialization(self):
         serialization = request_body.serialize(payload, content_type)
         self.assertEqual(
             serialization,
-            dict(body='?some_str=a&some_int=1&some_float=3.14')
+            dict(body='some_str=a&some_int=1&some_float=3.14')
```

Comment:
How about also adding a vlue which needs to be percent encoded
like some_str_with_space='hi there' -> 'hi%20there'